fetched/ssh_keys/sno

```diff
@@ -0,0 +1,49 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Pretty sure you shouldn't be committing SSH keys, even if they aren't directly security risks.